### PR TITLE
Update with resources and changes

### PR DESCRIPTION
**Pull request by mimizotti**
@emhickmann - Here are the changes to our Gear Up proposal w/ breakdowns, times, and resources. Thanks!

### CODE REVIEW

**Comment by mimizotti** [comment]
Tagging @dphilla @mikeyduece @nikborn

File: m4_sessions/1711-inning/Group_4.md [hunk 1]
```diff
@@ -13,13 +13,34 @@ Session Outcomes: What will students leave with or gain from this session?
...
-  * WHY - why it's important to write a technical blog post
-  * WHAT - topic of the blog
-  * HOW - how to write a technical blog (provide them a list of guidelines that will allow them to start writing that day)
-  * WHEN & WHERE - When and where to promote your blog.
+ * WHY it's important to write a technical blog post [MIMI] - 10 minutes
+  - Self-promotion
+  - Contribute/Give back to the community
+  - Provide examples of various Turing alumni blog posts & where they are now
+ * WHAT [MIKE] - 5 minutes
+  - High level: In-depth explanation of a technology, language, or algorithm
```
Comment (emhickmann) [inline m4_sessions/1711-inning/Group_4.md:21]: Really like this high, mid, and low.  Great opportunity for people to see where they can start to plug in to the blogs.

File: m4_sessions/1711-inning/Group_4.md [hunk 2]
```diff
@@ -13,13 +13,34 @@ Session Outcomes: What will students leave with or gain from this session?
...
+  - Mid-level: Implementation of a technology (for example: two-factor authentication)
+  - Low-level: Explanation on a function or piece of logic (for example, `this` in JavaScript or `self` in Ruby).
+ * HOW - How to turn an idea into an actionable outline [DAN] - 20 minutes
```
Comment (emhickmann) [inline m4_sessions/1711-inning/Group_4.md:24]: This is great!   Would love to see you add any resources to the markdown for this section.

File: m4_sessions/1711-inning/Group_4.md [hunk 3]
```diff
@@ -13,13 +13,34 @@ Session Outcomes: What will students leave with or gain from this session?
...
+* Students will be given time to outline and/or write
+* Break into small groups of 3-4 to go over their outline and receive feedback from peers
 
 Names of group members:
   * Mimi Le
   * Dan Phillips
   * Mike Heft
   * Nik Bornemeier
+  
+  Students should come to the session already completing the following:
```
Comment (emhickmann) [inline m4_sessions/1711-inning/Group_4.md:37]: Who is going to send the reminders?

File: m4_sessions/1711-inning/Group_4.md [hunk 4]
```diff
@@ -13,13 +13,34 @@ Session Outcomes: What will students leave with or gain from this session?
...
+  
+  * Come with 1-3 technical ideas
+  * Read: https://sendgrid.com/blog/write-technical-blog-posts/
+  * Read: https://quickleft.com/blog/write-technical-blog-post-part-1/
+  
+  **Remind students to bring their laptop, notebook and pen.**
+  
+  Rehearsal on January 2nd at 3pm.
```
Comment (emhickmann) [inline m4_sessions/1711-inning/Group_4.md:45]: YES!!  LOVE THAT YOU ARE DOING THIS!

File: m4_sessions/1711-inning/Group_4.md [hunk 5]
```diff
@@ -13,13 +13,34 @@ Session Outcomes: What will students leave with or gain from this session?
...
+ * WHAT [MIKE] - 5 minutes
+  - High level: In-depth explanation of a technology, language, or algorithm
+  - Mid-level: Implementation of a technology (for example: two-factor authentication)
+  - Low-level: Explanation on a function or piece of logic (for example, `this` in JavaScript or `self` in Ruby).
+ * HOW - How to turn an idea into an actionable outline [DAN] - 20 minutes
+ * WHEN & WHERE - When and where to promote your blog [NIK] - 10 minutes
+ 
+ACTIVITY TIME - 20 minutes
+* Students will be given time to outline and/or write
+* Break into small groups of 3-4 to go over their outline and receive feedback from peers
```
Comment (emhickmann) [inline m4_sessions/1711-inning/Group_4.md:29]: Would love to see you add a quick closing...maybe people could put on the board the things they are going to write about or put their outlines in the slack channel you start!

**Review by emhickmann — COMMENTED** [review]
Great overall session--be sure to check the comments on the closing!

File: m4_sessions/1711-inning/Group_4.md (hunk 3 above)
Comment (mimizotti) [reply m4_sessions/1711-inning/Group_4.md:37]: I'll be sending out reminders!

File: m4_sessions/1711-inning/Group_4.md (hunk 5 above)
Comment (mimizotti) [reply m4_sessions/1711-inning/Group_4.md:29]: Putting ideas up on the whiteboard is a great idea!

**Pull request merged**